javaparser-core/src/main/java/com/github/javaparser/ast/Node.java

```diff
@@ -65,11 +62,13 @@
 	public Node() {
 	}
 
+	public Node(Position begin, Position end) {
+		this.begin = begin;
+		this.end = end;
+	}
+
 	public Node(final int beginLine, final int beginColumn, final int endLine, final int endColumn) {
```

Comment:
Well if everybody is ok with it, I'd like to deprecate all the constructors of all the nodes with integer positions.